testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+        final Set<Integer> secondOne = Sets.newHashSet(11, 12, 13, 14, 15, 16);
+
+        Verify.assertSetsEqual(firstOne, secondOne);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test
+    public void pass_if_sets_are_null() {
+        Verify.assertSetsEqual(null, null);
+    }
+
+    @Test
+    public void pass_if_sets_are_equal() {
+        final Set<Integer> firstOne = Sets.newHashSet(1, 2, 3);
+        final Set<Integer> secondOne = Sets.newHashSet(firstOne);
+
+        Verify.assertSetsEqual(firstOne, secondOne);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_not_contains_entry() {
+        Verify.assertContainsEntry(1, 1, ArrayListMultimap.<Integer, Integer>create());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_is_null_in_contains_entry() {
+        Verify.assertContainsEntry(1, 1, null);
+    }
+
+    @Test
+    public void pass_if_multimap_contains_entry() {
+        final Integer entryKey = 1;
+        final Integer entryValue = 1;
+
+        final Multimap<Integer, Integer> multimap = ArrayListMultimap.create();
+        multimap.put(entryKey, entryValue);
+
+        Verify.assertContainsEntry(entryKey, entryValue, multimap);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_not_contains_key() {
+        Verify.assertContainsKey(1, Collections.emptyMap());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_contains_key() {
+        Verify.assertContainsKey(1, null);
+    }
+
+    @Test
+    public void pass_if_map_contains_key() {
+        final Integer key = 1;
+        Verify.assertContainsKey(key, Collections.singletonMap(key, 1));
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_contains_denied_key() {
+        final Integer key = 1;
+        Verify.denyContainsKey(key, Collections.singletonMap(key, 1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_deny_contains_key() {
+        Verify.denyContainsKey(1, null);
+    }
+
+    @Test
+    public void pass_if_map_not_contains_denied_key() {
+        Verify.denyContainsKey(1, Collections.emptyMap());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_not_contains_entry() {
+        final Integer key = 0;
+        Verify.assertContainsKeyValue(key, 0, Collections.singletonMap(key, 1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_contains_key_value() {
+        Verify.assertContainsKeyValue(1, 1, null);
+    }
+
+    @Test
+    public void pass_if_map_contains_entry() {
+        final Integer key = 1;
+        final Integer value = 1;
```

Comment:
Please use something else as a value to avoid a confusion by mistake. It's even better to use an object of a different type.